website/docs/support/github-101.md

```diff
@@ -0,0 +1,147 @@
+---
+id: github-101
+title: GitHub 101
+---
+
+Welcome to GitHub 101!
+
+This will be the guide to all the steps you need to follow
+in order to be able to participate in the [Farm Robotics Challenge](https://farmroboticschallenge.ai/).
+
+We use GitHub for all of our repositories and it is where you can access the files and code
+(hosted in repositories) you need to begin working on your project developing with the Amiga.
+
+## Create an account
+
+First you will need to have an Github account
+
+- To create one, just go to [github.com](https://www.github.com) and sign up!
+
+## Set up an ssh key
```

Comment:
I'd probably rename this section, as its content isn't necessarily just about creating an SSH key